BRANCHES.md

```diff
@@ -0,0 +1,46 @@
+# Maintained branches
+
+At any point in time, we have a number of maintained branches consisting of:
+
+- the development branch: this is where new features lands, as well as bug
+  fixes and security fixes
+- one or more LTS branches: these only get bug fixes and security fixes.
+
+We use [Semantic Versioning](https://semver.org/). In particular, we maintain
+API compatibility in the development branch between major version changes. We
+also maintain ABI compatibility within LTS branches; see the next section for
+details.
+
+## Backwards Compatibility
+
+If you have code that's working and secure with Mbed TLS x.y.z, then you
+should be able to re-compile it without modification with any later release
+x.y'.z' with the same major version number, and your code will still build, be
+secure, and work - unless it was relying on something that became insecure in
+the meantime (for example, crypto that was found to be weak). In case security
+comes in conflict with backwards compatibility, we will put security first,
+but always attempt to provide a compatibility option.
+
+For the LTS branches, additionally we try very hard to also maintain ABI
+compatibility (same definition as API except with re-linking instead of
```

Comment:
“same definition as API” but you didn't actually define “API compatibility”. Please add that expression to the previous paragraph where you define the concept.